pdf-rendering-template/bin/render-pdf.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * node.js script to render a given HTML file into the given PDF filename.
+ * Details of the API are at https://developers.google.com/web/tools/puppeteer/
+ *
+ * This command is invoked as:
+ *
+ *   $ node bin/render-pdf.js input.html output.pdf
+ *
+ */
+
+const puppeteer = require("puppeteer");
+
+const inputPath = `file://${process.argv[2]}`;
+const outputPath = process.argv[3];
```

Comment:
```suggestion

const [,, rawInputPath, outputPath] = process.argv;
const inputPath = `file://${rawInputPath}`;
```

🐘  It's better to use destructuring for index accessing, as it makes it easier to manage since changes to the order just need to happen in the one spot, rather than having to ensure the index is set correctly everywhere it's used.